.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+log4j2.xml
```

log4j2.xml we need)) commit it.
```suggestion
logs/*
```